articles/quickstart/native/ios-swift/_includes/_login_centralized.md

```diff
@@ -1,4 +1,10 @@
-<%= include('../../_includes/_getting_started', { library: 'Swift') %>
+<!-- markdownlint-disable MD002 MD041 -->
+
+## Before you Start
+
+This tutorial demonstrates how to add user login to a Swift application using Web Authentication with Auth0. If instead you're looking to implement [Sign In With Apple](https://developer.apple.com/sign-in-with-apple/) natively into your app, check out the [iOS Swift - Sign In With Apple tutorial](/quickstart/native/ios-swift-siwa).
```

Comment:
Could maybe follow the other style I mentioned of suggesting the alternative quickstart.